### ISSUE TITLE
Fix image roles in the banner

### ISSUE THREAD

**Issue by afeijoo** [uswds/uswds#3769]
- [ ] **Confirmed w Ammie and customer that this is not an issue and no changes are needed. Issue can be closed.**

### Describe the bug
Role=”img” does not have an accessible name 

### Steps to reproduce the bug
1. Go to https://designsystem.digital.gov/components/banner/
2. Click on the 3 images in the banner (building, lock, and lock)
3. Inspect source.
4. Note that the [role=”img”] is not a valid attribute on an <img> tag. 

### Expected behavior and expected behavior
See the [detailed report from Siteimprove](https://docs.google.com/document/d/1xiir1Boo59_ZH5uFtbOAtyU5PKUNHQ4OvPOAA5v-4jw/edit) that was submitted by a USWDS customer.

**Comment by mejiaj** [comment]
@jAigret-Bix @afeijoo I'm inclined to think this is a false positive with SiteImprove.

1. We verified before it was ignored and considered decorative without the role of `img`
1. The guidance linked above states:

    >In order for elements with a role of img be perceivable, authors MUST provide alternative text or a label determined by the accessible name calculation.

    On clicked the accessible name calculation it has:
    >One of the following values:
    >
    >author: name comes from values provided by the author in explicit markup features such as the aria-label attribute, the aria-labelledby attribute, or the host language labeling mechanism, such as the alt or title attributes in HTML, with HTML title attribute having the lowest precedence for specifying a text alternative.
    https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#namecalculation

Alt and title are listed there, but with lower specificity than aria-labelledby.

**Comment by afeijoo** [comment]
@jAigret-Bix @mejiaj & @thisisdano I responded to the customers who reported the initial issue and let them know it may be a false positive.

**Comment by afeijoo** [comment]
Confirmed that this is not an issue and no changes are needed.
